### DISCUSSION TITLE
Request - Temperature-

### DISCUSSION THREAD

**Discussion by Mazaud501** [Estia-advanced-programming/pandora-public#85] [Ideas for features]
Good day,

We have a proposal concerning the calculation of the temperature. Indeed, until now we have been doing the difference of the average temperature by the measured temperature. Here is the formula (double actualNoise = averageTemperature-flight.getRecords().get(i).getTemperature_in;). We think that the use of 2 variables should be in this order an we will obtain the right correction to apply for the temperature. Thus, we propose to go through (double actualNoise = flight.getRecords().get(i).getTemperature_in() - averageTemperature;) in order to obtain a more correct comparison of an input and an output and apply the corrections that they should make.

 Thanks, if you can give your opinion.

**Comment by dhmmasson** [comment]
We are talking about **milestone 4 Feature noiseTemp**.

I am not sure I understand your proposal...
```java
double actualNoise = averageTemperature-flight.getRecords().get(i).getTemperature_in;
```
```java
double actualNoise = flight.getRecords().get(i).getTemperature_in() - averageTemperature;
```

Do you mean we should do the the mean squared error (MSE) or mean squared deviation (MSD) instead of just the average error ?

**Comment by dhmmasson** [comment]
@WilliamDelamare Why are we assuming an average of 25ºC instead of using the actual average temperature to compute the noise?
